proto/umee/incentive/v1/genesis.proto

```diff
@@ -20,10 +20,9 @@ message GenesisState {
     (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
   ];
   repeated Bond              bonds               = 8 [(gogoproto.nullable) = false];
-  repeated PendingReward     pending_rewards     = 9 [(gogoproto.nullable) = false];
```

Comment:
Why this is removed?

Reply:
See PR description - it's an efficiency boost inspired by sdk staking logic